HoverBoardGigaDevice/Inc/defines_2-1-20.h

```diff
@@ -0,0 +1,35 @@
+#ifndef DEFINES_2_1_20_H
+#define DEFINES_2_1_20_H
+
+#define HALL_A PC13
+#define HALL_B PA1
+#define HALL_C PC14
+#define LED_RED PB4
+#define LED_GREEN PB3
+#define UPPER_LED PB2
+#define LOWER_LED PB5
+#define BUZZER PB9
+#define VBATT PA4
+#define CURRENT_DC PA0 
+#define SELF_HOLD	PB12
+#define BUTTON PA12
+
+// Brushless Control DC (BLDC) defines
+#define BLDC_GH PA10		// green	, Tommyboi2001 all bldc pins same as 2.0
+#define BLDC_GL PB15		
+#define BLDC_BH PA9			// blue
+#define BLDC_BL PB14		
+#define BLDC_YH PA8			// yellow
+#define BLDC_YL PB13		
+#define TIMER_BLDC_PULLUP	GPIO_PUPD_NONE	// robo: not sure if some boards indeed nned GPIO_PUPD_PULLUP like 2.2 or 2.3
+
+#define HAS_USART1	// tx=PA2,rx=PA3	uncomment to connect via 19200 baud serial
```

Comment:
I think there are pads for another USART but no connector. Maybe I'll need to solder something on if I want to connect the board to a micro controller and a slave